Makefile

```diff
@@ -0,0 +1,4 @@
+VERSION ?= 3.18.2
+
+dockertest-geospatial-amd64:
+	docker run --platform=linux/amd64 -it -v $(shell pwd):/app -v /tmp:/tmp --entrypoint=/bin/bash tiledbenterprise/notebook-python-r-julia-tilevcf-geo:$(VERSION) -c 'cd /app && python -m pip install --editable .[dev,tests] && python -m pytest $(OPTS)'
```

Comment:
Added for local testing purposes. It's potentially useful for all maintainers; the notebook image is one way to get an environment with geospatial libs and packages that aren't on PyPI or conda-forge.

Reply:
👍